src/themes/masschallenge/collections/menu.overrides

```diff
@@ -52,4 +44,23 @@
 	background-color: #fff;
 	background-size: 100px 137px, 100px 137px, 14px 137px, 14px 137px;
 	background-attachment: local, local, scroll, scroll;
-}
\ No newline at end of file
+}
+
+.ui-scrollable-menu .ui.dropdown, .ui-scrollable-menu .ui.menu .item{
```

Comment:
@freshprincekla the more semantic name would be `.ui.scrollable.menu`. Refer to the attached link. Since the scroll is part of the menu, it should describe the menu.
https://github.com/Semantic-Org/Semantic-UI/blob/deb275d2d5fe9a522a0b7bd8b6b6a1c939552718/src/definitions/collections/menu.less#L280